### PR TITLE
CC-33168 Angular 17 migration

### PR DESCRIPTION
**Pull request by supproduction**
### Overview

- Jira ticket: https://spryker.atlassian.net/browse/CC-33181

### CODE REVIEW

File: docs/dg/dev/upgrade-and-migrate/upgrade-to-angular-17.md [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+                "executor": "@nx/jest:jest",
+                "options": {
+                    "jestConfig": "frontend/merchant-portal/jest.config.ts"
+                }
+            }
+        }
+    }
+   ```
+
+   6. In `deploy.yml` update npm to `10` version.
```
Comment (tolerants) [inline docs/dg/dev/upgrade-and-migrate/upgrade-to-angular-17.md:440]: We also have to update requirement section of setup page for MP: https://docs.spryker.com/docs/dg/dev/frontend-development/202311.0/marketplace/set-up-the-merchant-portal.html#requirements

Also probably update https://docs.spryker.com/docs/dg/dev/system-requirements/202311.0/system-requirements.html#marketplace-system-requirements

File: docs/dg/dev/upgrade-and-migrate/upgrade-to-angular-17.md [hunk 2]
```diff
@@ -0,0 +1,474 @@
...
+
+## 2) Update Stylelint and css-loader
+
+Before starting the migration, make sure that stylelint and css-loader were updated by this doc: [Stylelint migration guide](/docs/dg/dev/upgrade-to-stylelint-16-and-css-loader-6.html) is required.
+
+## 3) Update npm dependencies
+
+1. In `package.json`, do the following:
+
+   1. Update or add the following dependencies and change commands:
```
Comment (tolerants) [inline docs/dg/dev/upgrade-and-migrate/upgrade-to-angular-17.md:76]:
```suggestion
   1. Update or add the following dependencies and change npm scripts:
```

Comment (tolerants) [reply docs/dg/dev/upgrade-and-migrate/upgrade-to-angular-17.md:76]: Maybe it makes sense to separate it into 3 separate sections, as long as for now it looks like a complete content for package.json. so something like this looks a bit better:

```
"scripts": {
    ...
    "mp:build": "nx build",
    "mp:build:watch": "nx build --watch",
    "mp:build:production": "nx build --configuration production",
    "mp:lint": "nx lint --no-eslintrc",
    "mp:test": "nx test",
    ...
},
```

**Review by tolerants — CHANGES_REQUESTED** [review]

File: docs/dg/dev/upgrade-and-migrate/upgrade-to-angular-17.md (hunk 1 above)
Comment (supproduction) [reply docs/dg/dev/upgrade-and-migrate/upgrade-to-angular-17.md:440]: Revert npm

File: docs/dg/dev/upgrade-and-migrate/upgrade-to-angular-17.md [hunk 3]
```diff
@@ -73,7 +73,7 @@ Before starting the migration, make sure that stylelint and css-loader were upda
 
 1. In `package.json`, do the following:
 
-   1. Update or add the following dependencies and change commands:
+   1. Change npm scripts::
```
Comment (tolerants) [inline docs/dg/dev/upgrade-and-migrate/upgrade-to-angular-17.md:76]: Not sure if `change` is the best word here. I'd use `update` or `adjust`.

**Review by tolerants — CHANGES_REQUESTED** [review]

**Review by tolerants — APPROVED** [review]

**Pull request merged**